### ISSUE TITLE
Compile on windows

### ISSUE THREAD

**Issue by BarhamAA** [Maroc-OS/CleanedBitCrack#16]
Hello, i tried to compile the code but the result was unsuccessful, please anyone compile the "EXE" version of the code for using on windows. i need CuBitCrack.exe and ClBitCrack.exe 

thank you all

**Comment by bodaciousbob** [comment]
yeah the windows build doesn't work

**Comment by Rynald0s** [comment]
Yup, windows built is effed up man! On build, we're missing /lib file for CudaKeySearchDevice .....
